python/feature_tokenization.py

```diff
@@ -161,7 +178,15 @@
 pat_inpat = [[inpat_dict[id] for id in np.unique(df.values)]
              for _, df in grouped_pat_keys]
 
-# Adding age at each visit
+# %% Adding the ICU indicator
+icu_pats = misa_data[misa_data.icu_visit == "Y"].pat_key
```

Comment:
@scotthlee The new targets file uses a string instead of a number for this column, so I had to swap this to populate it correctly